src/ucs/sys/topo/base/topo.c

```diff
@@ -415,11 +394,11 @@ ucs_topo_get_distance_sysfs(ucs_sys_device_t device1,
         return UCS_OK;
     } else if (ucs_topo_is_sys_root(common_path)) {
         if (ucs_topo_is_same_numa_node(device1, device2)) {
-            ucs_topo_common_numa_node_distance(distance);
+            *distance = ucs_global_opts.dist_node;
             return UCS_OK;
         }
 
-        ucs_topo_sys_root_distance(distance);
+        *distance = ucs_global_opts.dist_sys;
```

Comment:
*distance can contain AUTO value here (-2), how is it handled?

Reply:
Updated code to handle `auto` values.